.gitignore

```diff
@@ -2,3 +2,4 @@
 *.pem
 *.env
 *.ini
+!*.example
```

Comment:
To my knowledge, this line causes to ignore everything else apart from *.example in the repo root. However, the reason of this line ignore everything and keep *.example under the directory secrets. 
I believe that there should be couple of better ways to do this. 
Keywords to seek;
* Paths in .gitignore
* .gitignore per directory
* `git check-ignore` and `git status` commands

Relatively irrelevant bonus; glob matching in gitignore <https://git-scm.com/docs/gitignore>